script.service.ltvpl/resources/colours/Color.xml

```diff
@@ -0,0 +1,144 @@
+<colors>
```

Comment:
I didn't know this was supported in scripts. @ronie ?

Reply:
nope, it isn't.

Reply:
Removed